private_gpt/server/ingest/ingest_router.py

```diff
@@ -47,3 +47,15 @@ def list_ingested() -> IngestResponse:
     service = root_injector.get(IngestService)
     ingested_documents = service.list_ingested()
     return IngestResponse(object="list", model="private-gpt", data=ingested_documents)
+
+
+@ingest_router.delete("/ingest/{doc_id}", tags=["Ingestion"])
+def delete_ingested(doc_id: str) -> None:
+    """Delete the specified ingested Document.
+
+    The `doc_id` can be obtained from the `GET /ingest/list` endpoint
```

Comment:
This comment makes direct reference to the "local" setup, but PrivateGPT can be setup to work with different storage options (like remote vector and structured databases). I'd suggest to update this comment making it generic and talking about directories. Also, the doc is deleted not only from the document store. Let's make this more generic (check the rest of API docs)